server/src/app.ts

```diff
@@ -2,14 +2,16 @@
 //
 // SPDX-License-Identifier: MIT
 
+import cors from 'cors';
 import express, { Application } from 'express';
+
 import { router } from './routes/index';
-import cors from 'cors';
 
 export const app: Application = express();
 
 app.use(cors({
   origin: 'http://localhost:3005',
```

Comment:
instead of hardcoding, should we use the `FRONTEND_ORIGIN` env here?

Reply:
Yes, good point.